libs/langchain/langchain/document_loaders/srt.py

```diff
@@ -17,11 +20,30 @@ def __init__(self, file_path: str):
             )
         self.file_path = file_path
 
-    def load(self) -> List[Document]:
+    def load(self, duration: Optional[int] = None) -> List[Document]:
```

Comment:
same - should be an init param, not a load param

Reply:
Hello thank you for pointing out.

Reply:
acb6c8cda2cf594ad592df4c316340218b2ffc33 resolves this

Reply:
@efriis please review